docs/low-code-sql/development/gems/datasources.md

```diff
@@ -0,0 +1,92 @@
+| **1 Cluster and Fabric** - The Cluster and Fabric define the SQL Warehouse execution environment. Attach the desired Fabric which contains the Table of interest. |
+| **2 Click the Environment tab** - Browse the SQL Warehouse specified in the selected Fabric.                                                                      |
+| **3 Add a table** - Prophecy respects the user's permissions. Any table for which the user has read permission can be added to the Model canvas.                  |
+| **4 Source is added to the Model Canvas** - The Source named ORDERSHIPMENTS is now part of the Model, and is ready for transformation, joins, etc.                |
+| **5 Click the Project tab** - Lists all the Sources that are now accessible to any Model in the Project.                                                          |
+
+## Model
+
+A Model is similar to a Pipeline, as both contain data transformation steps. Unlike a Pipeline, however, each Model defines a single [materialized view or table](https://docs.getdbt.com/docs/build/materializations#materializations).
+
+Models are represented as visual format or as a single .sql file in `code` format. Because Models define a single materialized view or table, Models can serve as inputs to other models.
+
+![Model1](img/Model1.png)
+
+| Model Creation                                                                                                                                                |
+| ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **1 Open existing Model** - Within a Project, open an existing Model, eg MyNextModel.                                                                         |
+| **2 Add Model** - Click to add a new Model to the existing Model.                                                                                             |
+| **3 Model Name and File Path ** - Provide a name for the new Model, eg MyModel. Define the desired storage path in Git to store the new Model as a .sql file. |
+
+Now the new Model has been created and the canvas is displayed. Optionally, click `Config`. Prophecy makes it easy to decide whether a model should be materialized as a view, table, ephemeral, incremental, or dynamic table. Click [here](https://www.prophecy.io/blog/interactive-development) to see how to configure a table's materiliazation.
+
+![Model2](img/Model2.png)
+
+| Model as a Datasource                                                                                                    |
+| ------------------------------------------------------------------------------------------------------------------------ |
+| **4 Add a Source** - From the Environment tab, select a source for the new Model.                                        |
```

Comment:
Actually I can show a model that has already been created, to remove steps 4,5,6,7, and 8.

Reply:
I'll do this later? no time now